lib/Mojolicious/Command/daemon.pm

```diff
@@ -19,9 +19,11 @@ sub run {
     'k|keep-alive-timeout=i' => sub { $daemon->keep_alive_timeout($_[1]) },
     'l|listen=s'             => \my @listen,
     'p|proxy'                => sub { $daemon->reverse_proxy(1) },
-    'r|requests=i'           => sub { $daemon->max_requests($_[1]) };
+    'r|requests=i'           => sub { $daemon->max_requests($_[1]) },
+    't|trusted-proxy=s'      => \my @trusted;
```

Comment:
Could we reuse proxy instead? Something like `p|proxy:s{,1}`?

Reply:
Having just reverse proxy mode without trusted ips still has meaning, namely a simple "skip one level". Indeed -t still requires (and implicitly implies) -r because of how the scheme handling works. I can also imagine an extension to -r that would say how many levels to skip, like `-r 2`. All that just goes to say these aren't exactly the same concept and thus I don't want to conflate them at this outermost layer when they aren't internally.

Reply:
-r or -p..? -p is a boolean flag, which I can’t imagine need to be a supported IP address. That’s why I suggested keeping both. I haven’t tried {,} myself though, so I’m unsure how it works with zero arguments.

Reply:
With zero arguments an empty string is passes to the closure.

Reply:
Apologies, I meant -p not -r

Reply:
Fwiw. I don't see us adding `-p 2` when `-p 127.0.0.1` already works.